### PR TITLE
issue 646 adding auto-delete option to volume attachement in instance

### LINKED ISSUE #646: ibm_is_instance adding a data volume there is no option to auto-delete
### Affected Resource(s)
Please list the resources as a list, for example:
- ibm_is_instance
- ibm_is_volume

### Expected Behavior
The documentation does not indicate how to set the added volumes to automatically delete upon instance deletion.
image

### PR DESCRIPTION
**Pull request by deepaksibm**
Closes #646

### CODE REVIEW

**Comment by deepaksibm** [comment]
Screen Shot 2021-01-29 at 9 28 25 PM
Screen Shot 2021-01-29 at 9 13 02 PM
Screen Shot 2021-01-29 at 9 03 35 PM

**Comment by hkantare** [comment]
Add the documentaion update for this attribute as we dsicussed

File: website/docs/r/is_instance.html.markdown [hunk 1]
```diff
@@ -179,6 +179,8 @@ Nested `network_interfaces` block have the following structure:
   * `security_groups` - (Optional, list) Comma separated IDs of security groups.
   * `allow_ip_spoofing` - (Optional, bool) Indicates whether IP spoofing is allowed on this interface. If false, IP spoofing is prevented on this interface. If true, IP spoofing is allowed on this interface.
 * `volumes` - (Optional, list) Comma separated IDs of volumes. 
+* `auto_delete_volume` - (Optional, bool) If set to true, automatically deletes volumes attached to the instance.  
+**Note** Setting this argument may bring some inconsistency in volume resources since the volumes will be destroyed along with volumes.
```
Comment (hkantare) [inline website/docs/r/is_instance.html.markdown:183]: along with volumes.

shd 
along with instances.

Comment (deepaksibm) [reply website/docs/r/is_instance.html.markdown]: modified

**Pull request closed without merging**